syncserver-db-common/src/test.rs

```diff
@@ -1,6 +1,7 @@
 use diesel::{
     mysql::MysqlConnection,
     r2d2::{CustomizeConnection, Error as PoolError},
+    sqlite::SqliteConnection,
```

Comment:
I think that I can easily add a feature flag, at least for SQLite and MySQL.
I don't want to touch the code related to spanner because I can't test it.

Reply:
You may be able to feature flag a few things (like this `use`:

```rust
#[cfg(feature = "sqlite")]
use diesel::sqlite::SqliteConnection;
```
(and then flag the parts that include `SqliteConnection`)

For now, don't worry about it. As you get closer to the final version, we can work together to try and isolate things so that folk only include what they need. (This would include all the Spanner stuff, too.)